client/components/registration/Students.jsx

```diff
@@ -1,148 +1,18 @@
-          <label htmlFor='username' className='label'>Username</label>
-          <div className='control has-icons-left'>
-            <input id='username' className='input' name='username'
-              placeholder='username' onChange={this.handleChange} value={username} />
-            <span className='icon is-small is-left'>
-              <i className='fa fa-user' />
-            </span>
-          </div>
-        </div>
-
-        <div className='field'>
-          <label htmlFor='password' className='label'>Password</label>
-          <div className='control has-icons-left'>
-            <input id='password' className='input' name='password'
-              type='password' placeholder='password'
-              onChange={this.handleChange} value={password} />
-            <span className='icon is-small is-left'>
-              <i className='fa fa-key' />
-            </span>
-          </div>
-        </div>
-
-        <div className='field'>
-          <label htmlFor='confirm' className='label'>Confirm password</label>
-          <div className='control has-icons-left'>
-            <input id='confirm' className='input' name='confirm'
-              type='password' placeholder='confirm password'
-              onChange={this.handleChange} value={confirm} />
-            <span className='icon is-small is-left'>
-              <i className='fa fa-key' />
-            </span>
-            {showMatch && !match && <span style={this.styles.match}>*</span>}
-          </div>
-        </div>
-
-        <div className='field'>
-          <button className='button is-primary'
-            onClick={this.handleSubmit}>Register</button>
-        </div>
-      </form>
-    )
-  }
-
-  handleChange (e) {
-    const {name, value} = e.target
-    let match = this.state.match
-    match = name === 'password' ? value === this.state.confirm : match
-    match = name === 'confirm' ? value === this.state.password : match
-    this.setState({
-      [name]: value,
-      showMatch: this.state.showMatch || name === 'confirm',
-      match: match
-    })
-  }
-
-  handleSubmit (e) {
-    const {register} = this.props
-    const {username, password, confirm} = this.state
-    register(username, password, confirm)
-    e.preventDefault()
-  }
-}
-
-Students.propTypes = {
-  register: PropTypes.func
-}
-
-function mapDispatchToProps (dispatch) {
-  return {
-    register: (username, password, confirm) => {
-      if (password === confirm) {
-        dispatch(clearError())
-        return dispatch(register({username, password}))
-      }
-      dispatch(showError('Password and confirmation don\'t match'))
-    }
-  }
+// import {register} from '../../actions/auth'
+// import {showError, clearError} from '../../actions/error'
+
+const Students = (props) => {
+  return (
+    <form>
+      <div className='field'>
+        <button className='button is-primary GHlogin'><a href='http://localhost:3000/auth/github'>Continue with GitHub</a></button>
```

Comment:
* put the `<a>` on its own line
* see comment above about localhost:3000
* rename `GHlogin` CSS class to `gh-login`